ingestion_server/ingestion_server/es_mapping.py

```diff
@@ -51,109 +60,77 @@ def index_settings(table_name):
-                "type": "text",
-            },
-            "license_version": {
-                "type": "text",
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-            },
-            "license_url": {
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-                "type": "text",
-            },
-            "provider": {
-                "type": "text",
-                "fields": {"keyword": {"type": "keyword", "ignore_above": 256}},
-            },
-            "source": {
-                "fields": {"keyword": {"ignore_above": 256, "type": "keyword"}},
-                "type": "text",
+            # Rank feature fields
+            "standardized_popularity": {"type": "rank_feature"},
+            "authority_boost": {"type": "rank_feature"},
+            "authority_penalty": {
+                "type": "rank_feature",
+                "positive_score_impact": False,
             },
-            "filetype": {"type": "keyword"},
-            "created_on": {"type": "date"},
+            "max_boost": {"type": "rank_feature"},
+            "min_boost": {"type": "rank_feature"},
+            # Nested fields
             "tags": {
                 "properties": {
                     "accuracy": {"type": "float"},
+                    # Text-based fields
                     "name": {
                         "type": "text",
+                        "analyzer": "custom_english",
                         "fields": {
                             "keyword": {"type": "keyword", "ignore_above": 256},
                             "raw": {"type": "text", "index": True},
                         },
-                        "analyzer": "custom_english",
                     },
                 }
             },
-            "mature": {"type": "boolean"},
-            "standardized_popularity": {"type": "rank_feature"},
-            "authority_boost": {"type": "rank_feature"},
-            "authority_penalty": {
-                "type": "rank_feature",
-                "positive_score_impact": False,
-            },
-            "max_boost": {"type": "rank_feature"},
-            "min_boost": {"type": "rank_feature"},
-            "category": {"type": "keyword"},
-        }
+        },
     }
     media_properties = {
         "image": {
-            "aspect_ratio": {
-                "fields": {"keyword": {"type": "keyword"}},
-                "type": "text",
-            },
-            "size": {"fields": {"keyword": {"type": "keyword"}}, "type": "text"},
+            # Keyword fields
+            "aspect_ratio": {"type": "keyword"},
+            "size": {"type": "keyword"},
         },
         "audio": {
-            "bit_rate": {"type": "integer"},
-            "sample_rate": {"type": "integer"},
-            "genres": {"fields": {"keyword": {"type": "keyword"}}, "type": "text"},
-            "duration": {"type": "integer"},
```

Comment:
Wow, didn't realize we weren't using any of these!

Reply:
Yeah, there were ~8 fields that were being indexed but not used for searching or filtering (they were de-indexed in this PR).

Reply:
Isn't duration required for the FE **Duration** filter? 

<img src="https://cldup.com/YXBE1y3Xmd.png">

Reply:
That filter is actually on the `length` field of the `Audio` document computed here:
https://github.com/WordPress/openverse/blob/9a954958f542253021121d514d8ceb22814a980e/ingestion_server/ingestion_server/elasticsearch_models.py#L353-L359

Reply:
Nevermind! I remembered we used `length` (below) for this instead. It's all good (though it's a bit confusing).